twisted/scripts/_twistd_unix.py

```diff
@@ -198,6 +199,38 @@ def preApplication(self):
         self.oldstderr = sys.stderr
 
 
+    def _formatException(self, exception):
+        """
+        Format the C{exception} in preparation for writing to the
+        status pipe.  This does the right thing on Python 2 if the
+        exception's message is Unicode, and in all cases limits the
+        length of the message, prior to encoding, to 100 codepoints.
+
+        This means the return message may be longer than 100 bytes.
```

Comment:
Also, we don't have any tests for the exact limit of 100bytes.. so this is not part of the contract .